src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -33,7 +32,7 @@ import {getChannelListInfo} from '../../utils/string/StringUtils';
 
 const useCoreChatSystemHook = () => {
   const {localDb, refresh} = useLocalDatabaseHook() as UseLocalDatabaseHook;
-  const {anonProfileId, signedProfileId, profile} = useProfileHook();
+  const {anonProfileId, signedProfileId} = useProfileHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `profile` variable is no longer being destructured from `useProfileHook()`. If it's not used anywhere else in the code, this change is fine. However, if it's still being referenced elsewhere in the code, this will cause an error.
```diff
- const {anonProfileId, signedProfileId, profile} = useProfileHook();
+ const {anonProfileId, signedProfileId} = useProfileHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->